components/datepicker/stories/datepicker.stories.js

```diff
@@ -149,3 +149,108 @@ Range.args = {
 	isRange: true,
 	isOpen: false,
 };
+
```

Comment:
Not really related, but on [line 8](https://github.com/adobe/spectrum-css/blob/19c2b26639e6aad7ee834eeb5efa7f2aff1a872f/components/datepicker/stories/datepicker.stories.js#L8), I was curious about some capitalization. "text field" is not capitalized the first time it's used, but then capitalized in the next.

Is there a capitalizing rule for certain contexts, like when you're referring to the literal component, capitalize it because it's a name, but if you're speaking more generally, don't capitalize? I only ask because both are capitalized on the docs site.

Reply:
This is a great question. I know that Adobe uses sentence case, so that's why you see the components in the Storybook sidebar as "Date picker" instead of "Date Picker." I think in this case, we should not be capitalizing the component name so I'm going to adjust that.